src/vs/workbench/contrib/debug/browser/debugToolBar.ts

```diff
@@ -89,7 +90,7 @@ export class DebugToolBar extends Themable implements IWorkbenchContribution {
 		this.activeActions = [];
 		this.actionBar = this._register(new ActionBar(actionBarContainer, {
 			orientation: ActionsOrientation.HORIZONTAL,
-			actionViewItemProvider: (action: IAction) => {
+			actionViewItemProvider: (action: IAction, options: IBaseActionViewItemOptions) => {
 				if (action.id === FOCUS_SESSION_ID) {
 					return this.instantiationService.createInstance(FocusSessionActionViewItem, action, undefined);
```

Comment:
Should `options` be passed over here as well and to the action item below?

Reply:
This is a selectactionitem which does not support `IBaseActionViewItems`. This should be fine for the meantime.